configure.ac

```diff
@@ -705,6 +703,36 @@ case "$host" in
 		AC_MSG_RESULT([$blibflags])
 	fi
 	LDFLAGS="$saved_LDFLAGS"
+
+	AC_ARG_WITH([baslr],
```

Comment:
is there any reason this can't be part of the rest of the use_toolchain_hardening --with-hardening flags and be on by default when supported?